shadow-cljs.edn

```diff
@@ -18,4 +18,10 @@
                            :output-feature-set :es6}
         :modules {:main {:init-fn example.core/init}}
         :js-options {:resolve {"devcards-marked" {:target :npm :require "marked"}
-                               "devcards-syntax-highlighter" {:target :npm :require "highlight.js"}}}}}}
+                               "devcards-syntax-highlighter" {:target :npm :require "highlight.js"}}}}
+  :npm-dist {:target :npm-module
```

Comment:
Is this related to this benchmarks. If not please submit it as a separate PR. I know it's a pain, but I'd like to get in the habit of making PRs be about just 1 thing. This makes them easier to review and also makes it more reasonable to enforce squash merges since the final commit message will have a single theme and be revertable with no side effects.

Reply:
this is formatting I can remove